docs/manual/docs/cmd/outlook/report/report-getmailboxusagestorage.md

```diff
@@ -0,0 +1,40 @@
+# outlook report getmailboxusagestorage
+
+Get the amount of storage used in your organization. 
+
+## Usage
+
+```sh
+outlook report getmailboxusagestorage [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-p, --period <period>`|The length of time over which the report is aggregated. Supported values `D7|D30|D90|D180`
```

Comment:
Could you please change the `D7|D30|D90|D180` to use commas like `D7,D30,D90,180`? The `|` renders like a table in the markup causing the values to disappear. 

Probably you have taken this from another command, but we will have to change it to all the commands sometime in the future.

Reply:
Then we should change it in the other usage commands too.